### PR TITLE
TIDE-414 / TIDE-415 Remove scoring and other fixes.

### PR DESCRIPTION
**Pull request by rheinardkorf**
Additional ENV vars were required. Please see the README.md for
a list of ENV vars that needs to be specified for the Sync Server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,37 @@ to the `messages` channel.
...
+AWS_SQS_REGION=<AWS SQS REGION>
+AWS_SQS_KEY=<AWS SQS KEY>
+AWS_SQS_SECRET=<AWS SQS SECRET>
+AWS_SQS_QUEUE_NAME=<AWS SQS QUEUE NAME>
+  
+# TIDE API Settings
+# Has defaults
+TIDE_API_HOST=wptide.org
+TIDE_API_VERSION=v1
+TIDE_API_PROTOCOL=https
```
Comment (valendesigns) [inline README.md]: These are new, I like that this is now configurable. Nice!

File: cmd/sync-wp-repo/main.go [hunk 2]
```diff
@@ -209,9 +215,12 @@ func workerRequestThemeAudits(themes chan repo.Theme, db *scribble.Driver, messa
 func processTheme(theme repo.Theme, db *scribble.Driver, messages chan *queue.Task) error {
 	record := repo.Theme{} // Record schema for db.Read().
 
+	// Is this a forced sync?
+	forceSync := strings.ToLower(getEnv("SYNC_SERVER_FORCE_AUDITS", "no")) == "yes"
```
Comment (valendesigns) [inline cmd/sync-wp-repo/main.go]: To avoid duplication this should be set in `main()` and available at the package scope like `apiVersion` is.

Comment (rheinardkorf) [reply cmd/sync-wp-repo/main.go]: makes sense

**Comment by rheinardkorf** [comment]
@valendesigns over to you again

**Pull request merged**